src/service/tags/tags_repository.py

```diff
@@ -97,3 +99,57 @@ def get_tags(self, offset: int = 0, max_count: int = None) -> list:
         except Exception as error:
             self.logger.error(error)
             return []
+
+    def add_company_tag_query(self, tag_id: str, companies: list) -> str:
+        query = ""
+        for company_id in companies:
+            if company_id and company_id.strip():
+                company_tag_id = str(uuid.uuid4())
+                query += """
+                    INSERT INTO {table}
+                    VALUES ('{id}', '{tag_id}', '{company_id}');
+                """.format(
+                    table=TableNames.COMPANY_TAG,
+                    id=company_tag_id,
+                    tag_id=tag_id,
+                    company_id=company_id,
+                )
+        return query
+
+    def add_tag_query(self, tag_id: str, tag_name: str) -> str:
```

Comment:
Same here, the methods that return the queries shouldn't be without two underscores.